src/components/entries/entryQuickView.jsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import PropTypes from "prop-types";
+import YouTube from "../video/YouTube";
+import Sticky from "react-stickynode";
+import { Link } from "react-router-dom";
+import "./entryQuickView.css";
+
+const EntryQuickView = ({ entry, countries, closeQuickView }) => {
+  if (entry) {
+    let country = countries[entry.country_id];
+    return (
+      <aside id="pop-out">
+        <div id="quick-view">
+          <Sticky>
+            <header>
+              <img src={country.flag_url} />
+              <p className="p--closeX" onClick={closeQuickView}>
+                X
```

Comment:
Replace with String.fromCharCode(215) to get nice even 'x'.

Reply:
I didn't know that!